### PR TITLE
BD-2543 WhatsApp hidden page

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #BD-2543

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_hidden/private_betas/whatsapp_response_messaging.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+### Inbound messages
+
+Users can send you the following in their WhatsApp messages:
+
+Message feature    | Details
+----------- |---------------- 
+Text | 
+Images (JPEG and PNG)| Must be 8-bit, RGB or RGBA, and up to 5 MB for any type 
+Audio| audio/aac<br>audio/mp4<br>audio/mpeg<br>audio/amr<br>audio/ogg (only Opus Codecs, base audio/ogg is not supported)
+Documents | text/plain<br>application/pdf<br>application/vnd.ms-powerpoint<br>application/msword<br>application/vnd.ms-excel<br>application/vnd.openxmlformats-officedocument.wordprocessingml.document<br>application/vnd.openxmlformats-officedocument.presentationml.presentation<br>application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/whatsapp_response_messaging.md:199]:
```suggestion
Audio| audio/aac<br>audio/mp4<br>audio/mpeg<br>audio/amr<br>audio/ogg (only Opus Codecs, base audio/ogg is not supported)
Documents | text/plain<br>application/pdf<br>application/vnd.ms-powerpoint<br>application/msword<br>application/vnd.ms-excel<br>application/vnd.openxmlformats-officedocument.wordprocessingml.document<br>application/vnd.openxmlformats-officedocument.presentationml.presentation<br>application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
```
Do the details of these two rows need to be in specific formats?

File: _docs/_hidden/private_betas/whatsapp_response_messaging.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+You can send users the following in your WhatsApp messages:
+
+Message feature    | Details
+----------- |---------------- 
+Headers | 
+Text | Supports variable parameters
+Images (JPEG and PNG) | Must be 8-bit, RGB or RGBA, and up to 5 MB for any type 
+Body text | Supports variable parameters
+Footer text | Supports variable parameters 
+CTAs | See [Calls to actions](#ctas).
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/whatsapp_response_messaging.md:175]:
```suggestion
Headers | 
Text | Supports variable parameters.
Images (JPEG and PNG) | Must be 8-bit, RGB or RGBA, and up to 5 MB for any type.
Body text | Supports variable parameters.
Footer text | Supports variable parameters.
CTAs | See [Calls to actions](#ctas).
```
Should there be anything in the details for headers?

**Review by lydia-xie — APPROVED** [review]
lgtm! just flagged 2 things to review

File: _docs/_hidden/private_betas/whatsapp_response_messaging.md (hunk 2 above)
Comment (rachel-feinberg) [reply _docs/_hidden/private_betas/whatsapp_response_messaging.md:175]: That cell was left blank. Originally this info was provided in a bulleted list, and "Headers" was the only item that didn't have anything nested.

File: _docs/_hidden/private_betas/whatsapp_response_messaging.md (hunk 1 above)
Comment (rachel-feinberg) [reply _docs/_hidden/private_betas/whatsapp_response_messaging.md:199]: Ah, good catch!

**Pull request merged**